File: Arcade/Intro/02_Edge_of_the_Ocean/08_matrixElementsSum/main.py
def matrixElementsSum(matrix):
    totalSum = 0;
    dontConsid = [];
 
    for col in range(0, len(matrix[0])):
        for row in range(0, len(matrix)):
            if matrix[row][col] == 0:
                break;
            else:
                totalSum += matrix[row][col];
    return totalSum;

File: Arcade/Intro/02_Edge_of_the_Ocean/08_matrixElementsSum/main.ts
function matrixElementsSum(matrix: number[][]): number {
    let total: number = 0;
    for (let i: number = 0; i < matrix[0].length; i++) {
        for (let j: number = 0; j < matrix.length; j++) {
            if (matrix[j][i] === 0) {
                j = matrix.length;
            } else {
                total += matrix[j][i];
            }
        }
    }
    return total;
}
